corpus/backtick.txt

```diff
@@ -1,45 +1,83 @@
 ================================================================================
-Simple backtick
+simple backtick
 ================================================================================
 
-`foobar`
+a `foobar` b `:echo`
 
 --------------------------------------------------------------------------------
 
 (help_file
-  (line
-    (backtick
-      (word))))
+  (block
+    (line
+      (word)
+      (backtick
+        (word))
+      (word)
+      (backtick
+        (word)))))
 
 ================================================================================
-Backtick in text
+backtick in text
 ================================================================================
 
-Hello `world`, I am a markup language
+Hello `world`, I am `markup language`. But `this is
```

Comment:
The grammar supports same-line whitespace in a codespan, but I'm not sure if that's actually supported by vimdoc format.

Reply:
never mind this is fixed by inlining the `$.word2` cases into `$.word`. 🤷